### PR TITLE
Storage 4.9 rel notes

### PR DESCRIPTION
**Pull request by lpettyjo**
4.9

4.9 Release Notes storage content.

Preview: https://deploy-preview-36165--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html

PTAL: @openshift/storage, @jhou1

### CODE REVIEW

**Comment by sagidlow** [comment]
Looks to me overall. I think there are some issues with your file causes the build checks to fail. So make sure to fix that before merging.

**Comment by sagidlow** [comment]
Looks like there are two broken links that is causing the build failure.

**Comment by lpettyjo** [comment]
There are xrefs to two new CSI driver operator assembly files that doomed to fail until their two PRs are merged. Once merged, the this PR will not have this build fail issue.

**Comment by jhou1** [comment]
@duanwei33 @chao007 Can you PTAL?

**Comment by chao007** [comment]
Support some metrics for Local Storage Operator.
https://issues.redhat.com/browse/STOR-495

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -289,6 +289,66 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-9-storage"]
 === Storage
 
+[id="ocp-4-9-storage-aws-ebs-csi-ga"]
+==== Persistent storage using AWS EBS CSI driver operator is generally available
+OpenShift Container Platform is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
```
Comment (kalexand-rh) [inline release_notes/ocp-4-9-release-notes.adoc:294]:
```suggestion
{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
```

**Review by kalexand-rh — COMMENTED** [review]
I see one thing that you need to fix throughout the PR before we can merge this.

**Pull request merged**